src/main/java/com/hazelcast/cloud/ClientWithSsl.java

```diff
@@ -16,49 +14,46 @@
 import com.hazelcast.jet.pipeline.Sinks;
 import com.hazelcast.jet.pipeline.test.TestSources;
 import com.hazelcast.map.IMap;
-import com.hazelcast.spi.exception.TargetDisconnectedException;
 import com.hazelcast.sql.SqlResult;
 import com.hazelcast.sql.SqlRow;
 import com.hazelcast.sql.SqlService;
 
 /**
- * This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
+ * This is boilerplate application that configures client to connect Hazelcast Viridian cluster.
  * <p>
  * See: <a href="https://docs.hazelcast.com/cloud/java-client">https://docs.hazelcast.com/cloud/java-client</a>
  */
 public class ClientWithSsl {
 
     public static void main(String[] args) throws Exception {
+
+        // Configure the client.
         ClassLoader classLoader = ClientWithSsl.class.getClassLoader();
         Properties props = new Properties();
         props.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client.keystore").toURI().getPath());
         props.setProperty("javax.net.ssl.keyStorePassword", "YOUR_SSL_PASSWORD");
         props.setProperty("javax.net.ssl.trustStore",
-            classLoader.getResource("client.truststore").toURI().getPath());
+                classLoader.getResource("client.truststore").toURI().getPath());
         props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_SSL_PASSWORD");
         ClientConfig config = new ClientConfig();
         config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(props));
         config.getNetworkConfig().getCloudConfig()
-            .setDiscoveryToken("YOUR_CLUSTER_DISCOVERY_TOKEN")
-            .setEnabled(true);
+                .setDiscoveryToken("YOUR_CLUSTER_DISCOVERY_TOKEN")
+                .setEnabled(true);
         config.setProperty("hazelcast.client.cloud.url", "YOUR_DISCOVERY_URL");
         config.setClusterName("YOUR_CLUSTER_NAME");
 
+        config.getSerializationConfig().getCompactSerializationConfig().addSerializer(new CitySerializer());
+
+        System.out.println("Connect Hazelcast Viridian with TLS");
         HazelcastInstance client = HazelcastClient.newHazelcastClient(config);
         System.out.println("Connection Successful!");
 
         try {
-
-            mapExample(client);
-
-            //sqlExample(client);
-
-            //compactSerializationExample(client);
-
-            //nonStopMapExample(client);
-
-            //jetJobExample(client);
-
+            createMapping(client.getSql());
+            insertCities(client);
+            fetchCitiesWithSQL(client.getSql());
```

Comment:
minor: just to be consistent with the method above (which does not have withSQL suffix)
```suggestion
            fetchCities(client.getSql());
```